src/Core/Style.js

```diff
@@ -656,9 +674,11 @@ class Style {
      *
      * @param {Object} ctx - An object containing the feature context.
      *
-     * @return {string} The formatted string.
+     * @return {string|undefined} The formatted string if `style.text.field` is defined, nothing otherwise.
      */
     getTextFromProperties(ctx) {
+        if (!this.text.field) { return; }
```

Comment:
why you return `undefined` if there is no `this.text.field` ?

Reply:
I changed it so the method doesn't trigger an error when user set `Style.text` without setting `Style.text.field`.
For example, one could declare : 
```js
const layer = new LabelLayer('id', {
    source,
    domElement,
    style: new itowns.Style({
        text: { anchor : "top" },
    }),
};
```

Before this modification, setting `Style.text.field` was mandatory if you wanted to set some other properties of `Style.text`.

As I did not yet find a way of moving the anchor in the domElement css style, this part is unchanged. I will try to find a way of doing so.